python/doc/examples/functional_modeling/viscous_fall_field_function.ipynb

```diff
@@ -0,0 +1,353 @@
+      "text/html": [
+       "<TABLE><TR><TD></TD><TH>v0</TH></TR>\n",
+       "<TR><TD>0</TD><TD>0.0</TD></TR>\n",
+       "<TR><TD>1</TD><TD>0.12121212121212122</TD></TR>\n",
+       "<TR><TD>2</TD><TD>0.24242424242424243</TD></TR>\n",
+       "<TR><TD>3</TD><TD>0.36363636363636365</TD></TR>\n",
+       "<TR><TD>4</TD><TD>0.48484848484848486</TD></TR>\n",
+       "</TABLE>"
+      ],
+      "text/plain": [
+       "class=Sample name=Unnamed implementation=class=SampleImplementation name=Unnamed size=5 dimension=1 description=[v0] data=[[0],[0.121212],[0.242424],[0.363636],[0.484848]]"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "vertices = mesh.getVertices()\n",
+    "vertices[0:5]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Creation of the input distribution."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "distZ0 = ot.Uniform(100.0, 150.0)\n",
+    "distV0 = ot.Normal(55.0, 10.0)\n",
+    "distM = ot.Normal(80.0, 8.0)\n",
+    "distC = ot.Uniform(0.0, 30.0)\n",
+    "distribution = ot.ComposedDistribution([distZ0, distV0, distM, distC])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "4"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "dimension = distribution.getDimension()\n",
+    "dimension"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Then we define the Python function which computes the altitude at each time value. In order to compute all the altitudes with a vectorized evaluation, we first convert the vertices into a `numpy` `array` and use the `numpy` function `exp` and `maximum`: this increases the evaluation performance of the script. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def AltiFunc(X):\n",
+    "    g  = 9.81\n",
+    "    z0 = X[0]\n",
+    "    v0 = X[1]\n",
+    "    m  = X[2]\n",
+    "    c  = X[3]\n",
+    "    tau = m / c\n",
+    "    vinf = - m * g / c\n",
+    "    t = np.array(vertices)\n",
+    "    z = z0 + vinf * t + tau * (v0 - vinf) * (1 - np.exp( - t / tau))\n",
+    "    z = np.maximum(z,0.)\n",
+    "    return [[zeta[0]] for zeta in z]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In order to create a `Function` from this Python function, we use the `PythonPointToFieldFunction` class. The third argument, `outputDimension` is equal to `1`, because there is only one single output field i.e. the altitude. More precisely, if we had computed the speed as an extra output field, we would have set `2` instead."
```

Comment:
```suggestion
    "In order to create a `Function` from this Python function, we use the `PythonPointToFieldFunction` class. Since the altitude is the only output field, the third argument `outputDimension` is equal to `1`. If we had computed the speed as an extra output field, we would have set `2` instead."
```

Reply:
Fixed.